src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using Microsoft.PowerShell.MarkdownRender;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Show the VT100EncodedString or Html property of on console or show.
+    /// VT100EncodedString will be displayed on console.
+    /// Html will be displayed in default browser.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Show, "Markdown",
+        HelpUri = "TBD")]
+    [OutputType(typeof(string))]
+    public class ShowMarkdownCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Gets or sets InputObject of type Microsoft.PowerShell.MarkdownRender.MarkdownInfo to display.
+        /// </summary>
+        [ValidateNotNullOrEmpty]
+        [Parameter(Mandatory = true, ValueFromPipeline = true)]
+        public PSObject InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the switch to view Html in default browser.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter UseBrowser { get; set; }
+
+        private SteppablePipeline stepPipe;
+
+        /// <summary>
+        /// Override BeginProcessing.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            if (!this.MyInvocation.BoundParameters.ContainsKey("UseBrowser"))
```

Comment:
I'm not sure about this. I think it would miss `Show-Markdown -UseBrowser:$false`
what about `(bool)UseBrowser`

Reply:
Seems we could use `UseBrowser.IsPresent` .

Reply:
Fixed.